app/models/attraction.rb

```diff
@@ -12,7 +12,18 @@ def stale?
   end
 
   def self.all_trip_types
-    pluck(:trip_types).uniq
+    result = pluck(:trip_types)
+    .compact
+    .map { |types_string| JSON.parse(types_string) rescue [] }
+    .flatten
+    .uniq
```

Comment:
can change
```suggestion
    result = pluck(:trip_types)
    .compact
    .map { |types_string| JSON.parse(types_string) rescue [] }
    .flatten
    .uniq
```
result = pluck(:trip_types).uniq